__tests__/app.test.js

```diff
@@ -59,37 +59,76 @@ describe("app", () => {
     });
   });
   describe("/api/articles", () => {
-    it("status 200, responds with an array on object articles with the following properties:  author, title, article_id, topic, created_at, votes, article_img_url", () => {
+    it("feature request ## queries ## by topics = default all, sort_by any valid column = default date, order ASC || DESC = default DESC - defaults. Responds with an array of object articles", () => {
       return request(app)
         .get("/api/articles")
         .expect(200)
-        .then(({ body }) => {
-          expect(body.articles).toHaveLength(5);
-          body.articles.forEach((article) => {
-            expect(article).toEqual(
-              expect.objectContaining({
-                author: expect.any(String),
-                title: expect.any(String),
-                article_id: expect.any(Number),
-                topic: expect.any(String),
-                created_at: expect.any(String),
-                votes: expect.any(Number),
-                article_img_url: expect.any(String),
-              })
-            );
+        .then(({ body: articles }) => {
+          expect(articles).toHaveLength(5);
+          expect(articles[0]).toEqual({
+            author: "icellusedkars",
+            title: "Eight pug gifs that remind me of mitch",
+            article_id: 3,
+            topic: "mitch",
+            created_at: "2020-11-03T09:12:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            comment_count: "2",
```

Comment:
notice that the comment count is of string data type. something to be mindful of, and perhaps change to be INT (by editing your SQL query a little bit)